### ISSUE TITLE
loss.backward() is too slow

### ISSUE THREAD

**Issue by S-atan** [pyg-team/pytorch_geometric#9205]
### 🐛 Describe the bug

Hello, I replaced the ordinary convolution operation(Conv2d) with a graph convolution operation(GCNConv) for the skeleton joint point data. The graph convolution operation takes up to 20 seconds in the loss.backward() step, while the ordinary convolution operation only requires 0.2 seconds. Is there any method to accelerate this process?

### Versions

curl -OL https://raw.githubusercontent.com/pytorch/pytorch/main/torch/utils/collect_env.py
# For security purposes, please check the contents of collect_env.py before running it.
python3 collect_env.py
